### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by NoodleMage**
- Fixed some incorrect colors
- MyAnimeList card now looks correct.
- Empty view is now a custom view.

screener_20160516 153245
screener_20160516 153301

### CODE REVIEW

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -25,13 +25,14 @@
...
         <item name="windowActionModeOverlay">true</item>
-        <item name="actionModeBackground">?colorPrimaryDark</item>
+        <item name="actionModeBackground">?colorPrimary</item>
         <item name="actionBarTheme">@style/Theme.ActionBar</item>
         <item name="actionBarPopupTheme">@style/ThemeOverlay.AppCompat.Light</item>
         <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
 
         <!-- Custom Attributes-->
         <item name="selectable_list_drawable">@drawable/list_item_selector_light</item>
+        <item name="selectable_library_drawable">@drawable/library_item_selector_light</item>
```
Comment (inorichi) [inline app/src/main/res/values/themes.xml]: Isn't `<ripple>` for API >= 21?

Comment (NoodleMage) [inline app/src/main/res/values/themes.xml]: yes. but the drawable has an API v21+ version and a everything else version

Comment (inorichi) [inline app/src/main/res/values/themes.xml]: Right. Didn't look at the full path :)

File: app/src/main/java/eu/kanade/tachiyomi/widget/InformationView.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.RelativeLayout
+import eu.kanade.tachiyomi.R
+import eu.kanade.tachiyomi.util.getResourceColor
+import eu.kanade.tachiyomi.util.setVectorCompat
+import kotlinx.android.synthetic.main.view_information.view.*
+
+class InformationView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) :
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/widget/InformationView.kt]: How about calling it EmptyView? It makes some more sense IMO.

File: app/src/main/java/eu/kanade/tachiyomi/widget/EmptyView.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    init {
+        inflate(context, R.layout.view_empty, this)
+    }
+
+    /**
+     * Hide the information view
+     */
+    fun hide() {
+        this.visibility = View.INVISIBLE
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/widget/EmptyView.kt]: Have you tried `View.GONE`? I think it's better than this one

File: app/src/main/res/values/themes.xml [hunk 4]
```diff
@@ -25,13 +25,14 @@
 
         <!-- Themes -->
         <item name="windowActionModeOverlay">true</item>
-        <item name="actionModeBackground">?colorPrimaryDark</item>
+        <item name="actionModeBackground">?colorPrimary</item>
```
Comment (inorichi) [inline app/src/main/res/values/themes.xml]: Not sure about this one. IMO it should be different from the default toolbar color. For example, the gmail app has a totally different color when selection mode is enabled. But I agree `colorPrimaryDark` isn't any better.

Comment (NoodleMage) [inline app/src/main/res/values/themes.xml]: I liked the way it looked in Solid Explorer (also same color as toolbar). 
Will try to make it look like Google inbox one (white for normal theme, black for dark theme), still prefer the main color especially when you select chapter, the contrast isn't as big.

**Pull request merged**